WorkspaceView.py

```diff
@@ -4,6 +4,14 @@
 # *                                                                     *
 # ***********************************************************************
 
+# # TODO: the following try/except is a hack that only works while online for dev-testing
```

Comment:
To test, you might need to uncomment this code to import the new library. Actual add-on users will not have a problem as the Add-On Manager will honor the updates `package.xml` file.

Reply:
I've added a commit that simply imports `mistune`.

In the meantime you can install `mistune` in the following way: 

``` python
import Addon
from addonmanager_dependency_installer import DependencyInstaller

addon = Addon.Addon(
    name="Ondsel-Lens", 
    url="https://github.com/Ondsel-Development/Ondsel-Lens-Addon",
    branch="feature/LA-175-ondsel-start-tab",
)
dep_installer = DependencyInstaller([addon], ["mistune"], [])
dep_installer.run()
```
 
 Note that the branch may need to be `develop` after this PR has merged.